frontend/screens/MainScreen.js

```diff
@@ -1,30 +1,71 @@
 import { View, Text, Image, Dimensions, StyleSheet, Pressable, ScrollView } from 'react-native'
-import React from 'react'
+import React, {useEffect, useState} from 'react'
 import note from '../mocks/note'
 import ImagesCarousel from '../components/ImagesCarousel'
 import Button from '../components/Button'
 
+const getRandomNote = async (callback) => {
+    try {
+        const response = await fetch(
+            'http://192.168.1.91:8000/notes/getrandomnote',
+            );
+            const json = await response.json();
+            callback(json);
+        } catch (error) {
+            console.error(error);
+        }
+    };
+
+const getImages = async (callback, id) => {
+    try {
+        const response = await fetch(
+            `http://192.168.1.91:8000/notes/getimages/${id}`,
+            );
+            const json = await response.json();
+            callback(json.images);
+        } catch (error) {
+            console.error(error);
+        }
+    };
+
 const MainScreen = () => {
-    const images = note.images.map((imageURL) => {
+    const [note, setNote] = useState({})
+    const [images, setImages] = useState([])
+
+    useEffect(() => {
+            getRandomNote((json) => {
+                setNote(json);
+                getImages(setImages, json.id)})
+        },[]);
+    
+    
+    const imagesList = images?.map((image) => {
         return (
             <Image
-                key={imageURL}
+                key={image}
                 style={{ height: 300, width: 500 }}
-                source={{ uri: imageURL }}
+                source={{ uri: `data:image/png;base64,${image}` }} 
                 resizeMode='center'
             />
         )
     })
+    const labels = note.labels?.map((label) => {
+        return (
+            <Text
+                key={label.name}
```

Comment:
let's add some styling here to highlight labels of the note
Current state:
![image](https://github.com/ViktorLitviakov8050/randomNote/assets/98279909/25e413eb-16cc-471e-b82b-0a5dad5877ec)